packages/ddp-client/common/livedata_connection.js

```diff
@@ -80,6 +84,13 @@ export class Connection {
     // DDP.onReconnect.
     self.onReconnect = null;
 
+    // If a connection is forcefully closed (maxLength limit, for example) we don't
+    // want to trigger the methods again.
+    self.shouldRetryMethods = true;
+
+    // Store last error from method that broke the connection;
+    self.lastError = null;
```

Comment:
`disconnectError`, or `lastDisconnectError` might make more sense (more precise).
Also strictly speaking it's not necessarily "from method", this would be stored for any disconnect error.

Reply:
renamed